### DISCUSSION TITLE
Error: Create Pages site failed. AxiosError: Request failed with status code 403

### DISCUSSION THREAD

**Discussion by ooker777** [ObsidianPublisher/obsidian-github-publisher#137] [Q&A — Plugin] [answered]
I use the GitHub template. At the `Setup Pages` step it has this error:
```
Run actions/configure-pages@v2
  with:
    token: ***
    enablement: true
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.11.3/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.3/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.3/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.3/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.3/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.3/x64/lib
Warning: Get Pages site failed
Error: Create Pages site failed
Error: AxiosError: Request failed with status code 403
```
In my understanding this is due to a token issue. I don't know which token is here? I suppose not the one that the plugin uses to push to the repo, right? 

Here is [my repo](https://github.com/QuaCau-TheSphere/OPT-web)

**Comment by Mara-Li** [comment]
Are your settings like that : 
![image](https://user-images.githubusercontent.com/30244939/235358479-3b0a0eac-5c17-4287-bdd6-975577e8c337.png)

**Reply by Mara-Li** [reply, marked as answer]
btw, it not seems to be on my side, so maybe check the actions/configure-pages@v2 repo ?

**Reply by Mara-Li** [reply]
try to update the worfklows to v3 
https://github.com/actions/configure-pages/pull/50

**Reply by ooker777** [reply]
yeah I forgot to do that. Thanks for your quick response. I wonder how much time it takes to build it in your side? It's 15 min here and still not finished. I'm not sure if that's a healthy build time or not

**Reply by Mara-Li** [reply]
15min is not normal, how much file do you have ?

**Reply by ooker777** [reply]
529 files. It takes both GH pages and Netlify around 15 min for the building

**Reply by Mara-Li** [reply]
HUH. Yeah. Well.. Yeah. Dunno. I didn't try on so many file...
